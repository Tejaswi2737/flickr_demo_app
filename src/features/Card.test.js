import React from 'react';
import renderer from 'react-test-renderer';
import {
    DefaultCard,
} from './Card.stories';

it('renders correctly', () => {
    const tree = renderer.create(<DefaultCard {...DefaultCard.args} />)
        .toJSON();
    expect(tree).toMatchSnapshot();
});
