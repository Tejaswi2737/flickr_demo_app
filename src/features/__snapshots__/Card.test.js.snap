// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="sc-hKwDye eRTjdz"
>
  <a
    alt=""
    href="https://live.staticflickr.com/65535/51902520893_a1886b3020_m.jpg"
    rel="noreferrer"
    target="_blank"
  >
    <img
      alt="imgUrl"
      className="sc-dkPtRN kqsQTu"
      src="https://live.staticflickr.com/65535/51902520893_a1886b3020_m.jpg"
    />
  </a>
  <div
    className="sc-bdvvtL gOxqRk"
    color="light"
    height="medium"
    size="medium"
  >
    Image Title
  </div>
  <div
    className="sc-bdvvtL VPhdF"
    color="lightest"
    height="medium"
    size="medium"
  >
    24 Feb 2021
  </div>
  <div
    className="sc-gsDKAQ efWQeV"
    onClick={[Function]}
  >
    <div
      className="sc-bdvvtL kNTctQ"
      color="darker"
      height="medium"
      size="medium"
    >
      mountain
    </div>
  </div>
  <div
    className="sc-gsDKAQ efWQeV"
    onClick={[Function]}
  >
    <div
      className="sc-bdvvtL kNTctQ"
      color="darker"
      height="medium"
      size="medium"
    >
      potrait
    </div>
  </div>
  <div
    className="sc-gsDKAQ efWQeV"
    onClick={[Function]}
  >
    <div
      className="sc-bdvvtL kNTctQ"
      color="darker"
      height="medium"
      size="medium"
    >
      picture
    </div>
  </div>
  <div
    className="sc-gsDKAQ efWQeV"
    onClick={[Function]}
  >
    <div
      className="sc-bdvvtL kNTctQ"
      color="darker"
      height="medium"
      size="medium"
    >
      nixon
    </div>
  </div>
</div>
`;
